django_spanner/operations.py

```diff
@@ -301,6 +301,7 @@ def convert_datetimefield_value(self, value, expression, connection):
             value.second,
             value.microsecond,
         )
+        from django.conf import settings
```

Comment:
sorry but please undo this import fix, and add this to `docs/conf.py`

```
import django

sys.path.insert(0, os.path.abspath(".."))
os.environ["DJANGO_SETTINGS_MODULE"] = "tests.settings"
django.setup()
```

That should fix the issue docs build issue.